test_settings.py

```diff
@@ -88,6 +88,8 @@ def root(*args):
 
 LMS_ROOT_URL = "http://localhost:8000"
 
+OAUTH_ID_TOKEN_EXPIRATION = 60*60
```

Comment:
@e-kolpakov This setting [comes from edx-platform](https://github.com/edx/edx-platform/search?utf8=%E2%9C%93&q=OAUTH_ID_TOKEN_EXPIRATION), so I have to keep the same name.